state-sync/aptos-data-client/Cargo.toml

```diff
@@ -13,35 +13,32 @@ repository = { workspace = true }
 rust-version = { workspace = true }
 
 [dependencies]
-async-trait = "0.1.53"
-futures = "0.3.21"
-itertools = "0.10.0"
-rand = "0.7.3"
-serde = { version = "1.0.137", default-features = false }
-thiserror = "1.0.31"
-tokio = { version = "1.21.0", features = ["full"] }
-
-aptos-config = { path = "../../config" }
-aptos-crypto = { path = "../../crates/aptos-crypto" }
-aptos-id-generator = { path = "../../crates/aptos-id-generator" }
-aptos-infallible = { path = "../../crates/aptos-infallible" }
-aptos-logger = { path = "../../crates/aptos-logger" }
-aptos-metrics-core = { path = "../../crates/aptos-metrics-core" }
-aptos-time-service = { path = "../../crates/aptos-time-service", features = ["async"] }
-aptos-types = { path = "../../types" }
-
-netcore = { path = "../../network/netcore" }
-network = { path = "../../network" }
-storage-service-client = { path = "../storage-service/client" }
-storage-service-types = { path = "../storage-service/types" }
+aptos-config = { workspace = true }
```

Comment:
I don't know.. Is `aptos-config.workspace = true` cleaner?

Reply:
Yeah, it is. But `cargo sort` doesn't support it yet, e.g., see here: https://github.com/aptos-labs/aptos-core/blob/8958b0143286ed31f0924f5749f013b36999598a/scripts/rust_lint.sh#L34-L33. Once it does, we can string replace 😄